.github/workflows/tox.yml

```diff
@@ -181,4 +181,5 @@ jobs:
         if: ${{ contains(matrix.coverage, 'codecov') && matrix.pytest == 'true' }}
         uses: codecov/codecov-action@v3
         with:
-          token: ${{ secrets.CODECOV_TOKEN }}
+          file: ./coverage.xml
```

Comment:
Adding this line should be okay, however, if it's needed, it is a bug in codecov. 

We're always writing to `$GITHUB_WORKSPACE/coverage.xml` even if tox is running inside an isolated directory. This location should always be searched by default. https://github.com/OpenAstronomy/github-actions-workflows/blob/65452d98073d10f22caedc746d6e9a27e1a75320/tools/tox_matrix.py#L138

Reply:
I am really not sure what is happening upstream. Just following some ideas from https://github.com/codecov/codecov-action/issues/598